tests/31sync/04timeline.pl

```diff
@@ -16,10 +16,16 @@
          ( $room_id ) = @_;
 
          matrix_send_room_text_message( $user, $room_id,
-            body => "A test message",
+            body => "Test message 1",
          );
       })->then( sub {
-         ( $event_id ) = @_;
+         ( $event_id_1 ) = @_;
```

Comment:
The logic in these might read a little clearer, by way of associating the `$event_id_*` variables with their respective messages, by doing

       ...
       matrix_send_room_text_message( $user, $room_id,
          body => "Test message 1",
       )->on_done( sub { ( $event_id_1 ) = @_ } )
   })->then( sub {
       matrix_send_room_text_message( $user, $room_id,
          body => "Test message 2",
       )->on_done( sub { ( $event_id_2 ) = @_ } )
       ...